core/src/test/resources/sql/join.iq

```diff
@@ -854,4 +854,154 @@ EnumerableCalc(expr#0..1=[{inputs}], A=[$t0])
     EnumerableValues(tuples=[[{ 1 }, { null }]])
 !plan
 
+# Test for Double
+SELECT t1.a
```

Comment:
I think these are DECIMAL.
For DOUBLE you have to write 1.0e0

Reply:
the test is good, but add a DOUBLE test too
In general, joining on double is tricky, but for this case it should work fine

Reply:
the test for DECIMAL was good too, but it's also covered by the ARRAY tests below